python/tests/conftest.py

```diff
@@ -50,7 +50,6 @@
 
 BUCKET_ID = 0
 
-
```

Comment:
What's the reason for reverting this? Only doing it in `test_normalization.py` will mean most other test jobs will still fail (and the unit tests may fail before it gets to that file).

Reply:
The unittest on my branch fails regardless of the existence of setting of `MMAP_DEFAULT_SIZE` in `conftest.py` is. So at the moment I am fidgeting it.